### PR TITLE
Fixed movement bug :)

### PR DESCRIPTION
**Pull request by naslundx**

### CODE REVIEW

File: src/FGame.cpp [hunk 1]
```diff
@@ -73,7 +73,7 @@ void FGame::processEvents()
 	}
 	
 	// Check keyboard press
-	auto characters = m_engine->getCharacters();
+	auto &characters = m_engine->getCharacters();
```
Comment (naslundx) [inline src/FGame.cpp:76]: This was the problem, damn ampersand. :P

File: src/FObject.cpp [hunk 2]
```diff
@@ -123,9 +123,12 @@ void FObject::halt()
 
 void FObject::tick(float delta)
 {
+	m_velocity = m_velocity * 0.97f;
```
Comment (naslundx) [inline src/FObject.cpp:126]: Added a 3 % drag to cause things to slow down automatically.

File: src/FGame.cpp (hunk 1 above)
Comment (naslundx) [inline src/FGame.cpp:76]: @dekespo

Comment (dekespo) [inline src/FGame.cpp:76]: that's one sneaky :)

**Pull request merged**